src/channel/rpc_0_7_1.ts

```diff
@@ -44,7 +45,12 @@ const defaultOptions = {
 };
 
 export class RpcChannel {
-  readonly id = 'RPC07';
+  readonly id = 'RPC071';
+
+  /**
+   * This Channel provide communication using spec version
```

Comment:
```suggestion
   * RPC specification version this Channel class implements
```
This is a suggestion to sync the typedoc note with `this.specVersion`.